spec/Section 3 -- Type System.md

```diff
@@ -1370,6 +1373,50 @@ union SearchResult =
   | Person
 ```
 
+**Unions Implementing Interfaces**
+
+When defining unions that implement interfaces, each union member must
+explicitly implement each interface implemented by the union. For example, the
+member types of union SearchResult must each explicitly implement the Resource
+interface:
+
+```raw graphql example
+interface Resource {
+  url: String
+}
+
+union SearchResult implements Resource = Photo | Article
```

Comment:
[Typo]: You have `Photo` in the union definition and `Image` as the name of the defined type below.